packages/fast-components-react-base/src/checkbox/checkbox.tsx

```diff
@@ -87,6 +87,62 @@ class Checkbox extends Foundation<ICheckboxHandledProps & ICheckboxManagedClasse
 
         return super.generateClassNames(classes);
     }
+
+    /**
+     * Generates the correct attributes
+     */
+    private generateAttributes(): object {
+        const attributes: object = {};
+
+        /* tslint:disable:no-string-literal */
+        if (this.props.disabled) {
+            attributes["disabled"] = "disabled";
+        }
+
+        // If the author is not setting the checked state, we should let the checkbox manage itself
+        if (!isUndefined(this.props.checked)) {
+            if (this.props.onChange) {
+                attributes["checked"] = !!this.props.checked;
```

Comment:
you shouldn't need to use string accessors anymore - this was a work-around that stemmed from not having accurate typings in the other repository.